Dockerfile

```diff
@@ -27,6 +27,15 @@ RUN pip install -U pip \
     && rm -rf /tmp/*
 
 COPY ./dags ${AIRFLOW_HOME}/dags
+
+# add maintenance dags
+ADD https://raw.githubusercontent.com/teamclairvoyant/airflow-maintenance-dags/master/clear-missing-dags/airflow-clear-missing-dags.py \
```

Comment:
Can we do this in one layer using:
```
ADD [--chown=<user>:<group>] ["<src>",... "<dest>"]
```
ref: https://docs.docker.com/engine/reference/builder/#add

Reply:
updated.